Makefile

```diff
@@ -13,7 +13,7 @@ IMAGE_TAG_PREFIX ?=
 
 ## dockerfile for each container default to alpine base image
 DOCKERFILE_BACKEND ?= Dockerfile.backend-container
-DOCKERFILE_FRONTEND ?= Dockerfile.frontend-container
+DOCKERFILE_FRONTEND ?= frontend/Dockerfile.frontend-container
 
 BINARIES=tornjak-backend tornjak-manager
 IMAGES=$(BINARIES) tornjak-frontend 
```

Comment:
"tornjak-frontend " -> "frontend"

Reply:
i believe the images should be labeled `tornjak-frontend` to make it clear this is tornjak component, as opposed to some other `frontend` image there might be.